src/adServerManager.js

```diff
@@ -0,0 +1,90 @@
+import { getGlobal } from 'src/prebidGlobal';
+
+const prebid = getGlobal();
+
+/**
+ * This file serves as the plugin point for adserver module functionality.
+ *
+ * Ad Server modules should call {@link registerAdserver} to connect into prebid-core.
+ *
+ * If publishers bundle Prebid with a single Ad Server module, they'll be able to interact with it
+ * through the properties and functions on `pbjs.adserver`.
+ *
+ * If publishers bundle with more than one Ad Server, they'll need to interact with them through
+ * `pbjs.adservers.{adServerCode}`, where the adServerCode is defined by the ad server itself.
```

Comment:
I'd image the pub would be configured with an Ad Server definition (include "none") and an AdUnit could be configured with an override to use, if present.